CesiumAsync/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ target_include_directories(
         ${CMAKE_CURRENT_LIST_DIR}/include
     PRIVATE
         ${CMAKE_CURRENT_LIST_DIR}/src/
+        ${CMAKE_CURRENT_LIST_DIR}/../extern/sqlite3/
```

Comment:
I'm not clear on why this is needed. Shouldn't the dependency on the (aliased) sqlite3 via `target_link_library` automatically add sqlite3's include directory? Just like it does in the non-aliased case?

Reply:
It was a side effect of skipping the install command. Once it was using the aliased version it was no longer looking into the extern/sqlite directory. Only the conan version.